client/client_grpc_collection.go

```diff
@@ -394,3 +395,46 @@ func (c *grpcClient) ReleaseCollection(ctx context.Context, collName string) err
 	}
 	return handleRespStatus(resp)
 }
+
+// GetReplicas gets the replica groups as well as their querynodes and shards information
+func (c *grpcClient) GetReplicas(ctx context.Context, collName string) ([]*entity.ReplicaGroup, error) {
+	if c.service == nil {
+		return nil, ErrClientNotReady
+	}
+	coll, err := c.ShowCollection(ctx, collName)
+	if err != nil {
+		return nil, err
+	}
+
+	req := &server.GetReplicasRequest{
+		CollectionID:   coll.ID,
+		WithShardNodes: true, // return nodes by default
+	}
+
+	resp, err := c.service.GetReplicas(ctx, req)
+	if err != nil {
+		return nil, err
+	}
+	if err = handleRespStatus(resp.GetStatus()); err != nil {
+		return nil, err
```

Comment:
Please add unit test to cover server return error status.

Reply:
Done